int8 ec_id          # ethercat板子的索引
int8 slave_id       # ethercat板子的子板的索引
int8 passage        # 通道号
int8 motor_id       # 电机id
int16 cur_tor       # 电机力矩
int8 ctrl_status    # 控制状态
int8 ack_status     # 返回状态
